data/campaigns/Under_the_Burning_Suns/scenarios/02_Across_the_Harsh_Sands.cfg

```diff
@@ -703,38 +703,40 @@
             [/have_unit]
         [/filter_condition]
 
-        [message]
-            speaker=Kaleh
-            message= _ "That’s the last of them. This looks like a hunting party, they must have a camp around here somewhere."
-        [/message]
+        [kill]
+            role="Hunting Ogre"
+        [/kill]
+
+        [redraw]
+        [/redraw]
 
         [item]
             x=$x1
             y=$y1
             image=items/potion-grey.png
         [/item]
 
-        [redraw]
-        [/redraw]
+        [message]
+            speaker=Kaleh
+            message= _ "That’s the last of them. This looks like a hunting party, they must have a camp around here somewhere."
+        [/message]
 
         [message]
             speaker=Nym
             message=_"Hey! Look there, they dropped something... A stone bottle, sealed. I wonder what’s inside..."
         [/message]
 
-        # Move Nym adjacent to the bottle. During a die event, $unit is still blocking the hex, so
-        # Nym probably ends up running to the hex north-west of $x1,$y1.
-        #
-        # Don't move her if she's already adjacent to it - if she landed the killing blow, moving
-        # her now means that she doesn't get the XP. Also, the unit still has ZoC, and she'll take a
-        # longer route to avoid it, which looks particularly obvious if Nym was adjacent the ogre.
+        # Move Nym adjacent to the bottle, but don't move her if she's already adjacent to it
```

Comment:
Move Nym <strike>adjacent</strike> to the bottle, but ...

As the ogre is no longer blocking the hex, `{MOVE_UNIT id=Nym $x1 $y1}` now puts her on the same hex as the bottle. The effect looks good, this is just a request to update the comment.